src/simple_cache_client.rs

```diff
@@ -17,17 +17,11 @@ use crate::endpoint_resolver::MomentoEndpointsResolver;
 use crate::{grpc::header_interceptor::HeaderInterceptor, utils::connect_channel_lazily};
 
 use crate::response::{
-    cache_dictionary_fetch_response::{
-        MomentoDictionaryFetchResponse, MomentoDictionaryFetchStatus,
-    },
-    cache_dictionary_get_response::{MomentoDictionaryGetResponse, MomentoDictionaryGetStatus},
-    cache_dictionary_set_response::{MomentoDictionarySetResponse, MomentoDictionarySetStatus},
-    cache_get_response::{MomentoGetResponse, MomentoGetStatus},
-    cache_set_response::{MomentoSetResponse, MomentoSetStatus},
-    create_signing_key_response::MomentoCreateSigningKeyResponse,
-    error::MomentoError,
-    list_cache_response::{MomentoCache, MomentoListCacheResult},
-    list_signing_keys_response::{MomentoListSigningKeyResult, MomentoSigningKey},
+    MomentoCache, MomentoCreateSigningKeyResponse, MomentoDictionaryFetchResponse,
+    MomentoDictionaryFetchStatus, MomentoDictionaryGetResponse, MomentoDictionaryGetStatus,
+    MomentoDictionarySetResponse, MomentoDictionarySetStatus, MomentoError, MomentoGetResponse,
+    MomentoGetStatus, MomentoListCacheResult, MomentoListSigningKeyResult, MomentoSetResponse,
+    MomentoSetStatus, MomentoSigningKey,
 };
```

Comment:
If it was up to me I'd just convert this to a glob import but I'm not entirely sure if we're ok with doing that in this codebase.

Reply:
> If it was up to me I'd just convert this to a glob import but I'm not entirely sure if we're ok with doing that in this codebase.

We don't really do glob imports. It's a small character count savings trading for symbol location ambiguity. It is sad for people who are reading without an IDE.

Reply:
👍

That's why I checked :)